server/modules/freckle/projects.js

```diff
@@ -27,13 +29,28 @@ export const getOrCreateProject = async (projectName) => {
       name,
     ));
     // write automation success to database
+    await createOrUpdateFreckleAutomation({
```

Comment:
What if this function returns before it gets here?

Reply:
Fixed it